SamplePay/app/src/main/java/com/example/android/samplepay/PaymentActivity.kt

```diff
@@ -32,21 +42,40 @@ import androidx.lifecycle.observe
  * is disabled unless the calling app is Chrome.
  */
 class PaymentActivity : AppCompatActivity() {
-
+    private lateinit var mErrorMessage: String
```

Comment:
`mErrorMessage`
We usually don't prefix member fields in Kotlin.

Reply:
Done, and everywhere else in the code.